mopro-core/src/middleware/gpu_exploration/msm_bench.csv

```diff
@@ -0,0 +1,35 @@
+num_msm,avg_processing_time(sec),total_processing_time(sec),memory_allocated(MiB)
```

Comment:
This probably belongs in a different non-code folder

maybe `mopro-core/benchmarks/gpu_explorations`?

Reply:
Exactly, thanks for the suggestion